extensions/wikia/UserLogin/js/MarketingOptIn.js

```diff
@@ -12,21 +12,21 @@ define('usersignup.marketingOptIn', ['wikia.geo'], function (geo) {
 		var $optInField = wikiaForm.inputs.wpMarketingOptIn,
 			$optInGroup = wikiaForm.getInputGroup('wpMarketingOptIn'),
 			isEurope = geo.getContinentCode() === 'EU',
-			isCanada = geo.getCountryCode() === 'CA';
+			isCanada = geo.getCountryCode() === 'CA',
+			isJapan = geo.getCountryCode() === 'JP';
```

Comment:
Instead of creating new variable for new country, what do you think about changing approach to ```enabled``` and ```isChecked```? This way we'll manipulate on array of countries instead of whole method.